### PR TITLE
dumpling/tests: update sync_diff configs for CI

### LINKED ISSUE #29388: dumpling: update sync_diff configs for CI
The sync_diff_inspector's config has been updated to v2. The configs used by dumpling need to be updated too.

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve?

Issue Number: close #29388

Problem Summary:

configs of sync_diff_inspector used by dumpling is outdated.

### What is changed and how it works?

update configs

### Release note

```release-note
None
```

### CODE REVIEW

File: dumpling/tests/all_generate_column/conf/diff_config.toml [hunk 1]
```diff
@@ -1,40 +1,28 @@
...
 
-sample-percent = 100
+export-fix-sql = true
 
-use-checksum = true
+check-struct-only = false
 
-fix-sql-file = "fix.sql"
+[task]
+    output-dir = "./output"
```
Comment (lichunzhu) [inline dumpling/tests/all_generate_column/conf/diff_config.toml:10]: Can we print the result in dumpling's test dir `$DUMPLING_OUTPUT_DIR`? To make it easier to track in the future.

**Review by lichunzhu — APPROVED** [review]
Rest LGTM

**Review by Ehco1996 — COMMENTED** [review]
need add a empty line at end of file
rest lgtm

**Review by Ehco1996 — APPROVED** [review]

**Pull request merged**